plugins/contiv/plugin_impl_contiv.go

```diff
@@ -90,9 +90,11 @@ type Config struct {
 
 // OneNodeConfig represents configuration for one node. It contains only settings specific to given node.
 type OneNodeConfig struct {
-	NodeName           string
-	MainVppInterface   InterfaceWithIP
-	OtherVPPInterfaces []InterfaceWithIP // other configured interfaces get only ip address assigned in vpp
+	NodeName           string            // name of the node, should match withs the hostname
+	MainVppInterface   InterfaceWithIP   // main VPP interface used for the inter-node connectivity
+	OtherVPPInterfaces []InterfaceWithIP // other interfaces on VPP, not necessarily used for inter-node connectivity
+	Gateway            string            // IP address of the default gateway
+	StealHostInterface string            // interface in the host Linux (e.g. eth0) to be stolen and connected to VPP
```

Comment:
shouldn't StealHostInterface be part of another PR?

Reply:
right, is an future option. Removed.